BuildRadiator/Client/Controllers/DashboardController.js

```diff
@@ -9,49 +9,93 @@
     ctrl.committerLimit = 11;
 
     function onMessageUpdate( message ) {
-      var tile = ctrl.tiles.find( function( tile ) {
+      var tiles = ctrl.tiles.filter( function( tile ) {
```

Comment:
Updated to use `filter` in case the same message tile was placed multiple times on the same page. Not used in our set up but could be useful for the future.